### PR TITLE
fix: misc vault fixes 5

### PR DESCRIPTION
**Pull request by tyleroooo**

### CODE REVIEW

File: src/pages/vaults/VaultInfoSections.tsx [hunk 1]
```diff
@@ -105,21 +105,38 @@ const $DetailCard = styled.div`
...
-              {stringGetter({ key: STRING_KEYS.LEARN_MORE })}
-            </Link>
-          ),
-        },
-      })}
+    <div className={className} tw="flex flex-col gap-0.5 text-color-text-0 font-small-medium">
+      <p>
+        {stringGetter({
+          key: STRING_KEYS.VAULT_DESCRIPTION,
+        })}{' '}
```
Comment (moo-onthelawn) [inline src/pages/vaults/VaultInfoSections.tsx:115]: nit: can we get rid of the empty space

File: src/pages/vaults/VaultInfoSections.tsx [hunk 2]
```diff
@@ -105,21 +105,38 @@ const $DetailCard = styled.div`
...
+          </Link>
+        )}
+      </p>
+      <p>
+        {stringGetter({
+          key: STRING_KEYS.VAULT_OPERATOR_DESCRIPTION,
+          params: {
+            OPERATOR_NAME: operatorName,
+          },
+        })}{' '}
```
Comment (moo-onthelawn) [inline src/pages/vaults/VaultInfoSections.tsx:128]: and here

File: src/pages/vaults/VaultPositionsTable.tsx [hunk 3]
```diff
@@ -128,7 +128,9 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
                   type={OutputType.Asset}
                   tag={
                     <$Label>
-                      {getDisplayableAssetFromBaseAsset(marketsData[marketId ?? '']?.assetId)}
+                      {marketId === USDC_MARKET_HARDCODED
+                        ? 'USDC'
```
Comment (moo-onthelawn) [inline src/pages/vaults/VaultPositionsTable.tsx:132]: nit instead of hardcoding can we use the output of `useTokenConfigs`

**Review by moo-onthelawn — APPROVED** [review]

File: src/pages/vaults/VaultInfoSections.tsx (hunk 1 above)
Comment (tyleroooo) [reply src/pages/vaults/VaultInfoSections.tsx:115]: I either need a space or to add margin. Space feels better so it is consistent with surrounding text.

File: src/pages/vaults/VaultTransactions.tsx [hunk 4]
```diff
@@ -89,9 +89,10 @@ export const VaultTransactionsTable = ({
                   value={timestampMs}
                   type={OutputType.Date}
                   dateOptions={{ format: 'medium' }}
+                  title=""
                 />
                 <div tw="text-[0.75rem] leading-[0.7rem] text-color-text-0">
-                  <Output value={timestampMs} type={OutputType.Time} timeOptions={{}} />
+                  <Output value={timestampMs} type={OutputType.Time} timeOptions={{}} title="" />
```
Comment (jaredvu) [inline src/pages/vaults/VaultTransactions.tsx:95]: did you mean to add a title?

**Review by jaredvu — APPROVED** [review]

File: src/pages/vaults/VaultTransactions.tsx (hunk 4 above)
Comment (tyleroooo) [reply src/pages/vaults/VaultTransactions.tsx:95]: nope, want it removed for these

**Pull request merged**